x/posts/internal/types/post.go

```diff
@@ -9,18 +9,18 @@ import (
 )
 
 // ---------------
-// --- Post id
+// --- Post newID
 // ---------------
 
-// PostID represents a unique post id
+// PostID represents a unique post newID
 type PostID uint64
 
-// Valid tells if the id can be used safely
+// Valid tells if the newID can be used safely
```

Comment:
I think this is a refactoring error

```suggestion
// Valid tells if the id can be used safely
```